node/nodeHelper.go

```diff
@@ -177,6 +177,7 @@ func CreateNode(
 	}
 
 	esdtNftStorage, err := builtInFunctions.NewESDTDataStorage(builtInFunctions.ArgsNewESDTDataStorage{
+		// TODO: Check if this is all right (not passing AccountsAdapter).
```

Comment:
Removed TODO. I was wondering if `builtInFunctions.NewESDTDataStorage` should be passed `stateComponents.AccountsAdapter()` instead of the one suited for resolving API calls, `stateComponents.AccountsAdapterAPI()`.